Dockerfile

```diff
@@ -1,10 +1,38 @@
-FROM konstin2/meine-stadt-transparent:v0.2.5
+ARG MST_SOURCE_IMAGE=konstin2/meine-stadt-transparent:v0.2.6
+FROM ${MST_SOURCE_IMAGE} AS source
+FROM node:10 AS front-end
 
-ENV TEMPLATE_DIRS=customization/templates
+ENV NODE_ENV=production \
+  TEMPLATE_MAIN_CSS=mainapp-muenster
+WORKDIR /app
 
-COPY --chown=www-data customization /app/customization
+COPY --from=source /app/package.json /app/package.json
+COPY --from=source /app/package-lock.json /app/package-lock.json
 
-# compile translations etc
+RUN npm ci --dev
+
+COPY --from=source /app/etc /app/etc
+COPY customization /app/customization
+COPY --from=source /app/mainapp/assets /app/mainapp/assets
+RUN npm run build:prod
+
+FROM ${MST_SOURCE_IMAGE}
+
+ENV TEMPLATE_DIRS=customization/muenster/templates \
+  TEMPLATE_MAIN_CSS=mainapp-muenster
+
+COPY --chown=www-data --from=front-end /app/customization /app/customization
+COPY --chown=www-data --from=front-end /app/mainapp/assets /app/mainapp/assets
+
+# Copy /static from the source, but change the owner.
+# Otherwise collectstatic in the next step fails because
+# in the original container /static is owned by root:root
```

Comment:
That's odd, I get the following:
```
$ docker run --rm -it --entrypoint ls konstin2/meine-stadt-transparent -lah /static
total 68K
drwxr-xr-x 1 www-data www-data 4.0K May  7 16:09 .
drwxr-xr-x 1 root     root     4.0K May 12 19:23 ..
drwxr-xr-x 6 www-data www-data 4.0K May  7 16:09 admin
drwxr-xr-x 2 www-data www-data 4.0K May  7 16:09 build
drwxr-xr-x 4 www-data www-data 4.0K May  7 16:09 bundles
drwxr-xr-x 2 www-data www-data 4.0K May  7 16:09 css
drwxr-xr-x 2 www-data www-data 4.0K May  7 16:09 email
drwxr-xr-x 2 www-data www-data 4.0K May  7 16:09 images
drwxr-xr-x 3 www-data www-data 4.0K May  7 16:09 js
drwxr-xr-x 6 www-data www-data 4.0K May  7 16:09 wagtailadmin
drwxr-xr-x 4 www-data www-data 4.0K May  7 16:09 wagtaildocs
drwxr-xr-x 3 www-data www-data 4.0K May  7 16:09 wagtailembeds
drwxr-xr-x 4 www-data www-data 4.0K May  7 16:09 wagtailimages
drwxr-xr-x 3 www-data www-data 4.0K May  7 16:09 wagtailsearch
drwxr-xr-x 3 www-data www-data 4.0K May  7 16:09 wagtailsnippets
drwxr-xr-x 4 www-data www-data 4.0K May  7 16:09 wagtailusers
drwxr-xr-x 5 www-data www-data 4.0K May  7 16:09 web
```

Reply:
I think this is maybe related to the changes you‘ve made to the Dockerfile in the last release? I am certain without this, collectstatic failed with missing permissions.